dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -325,35 +292,35 @@ export const AdSlot: React.FC<Props> = ({ position, display }) => {
 						data-link-name="ad slot comments"
 						data-name="comments"
 						data-mobile={[
-							`${Size.outOfPage}`,
-							`${Size.empty}`,
-							`${Size.halfPage}`,
-							`${Size.outstreamMobile}`,
-							`${Size.mpu}`,
-							`${Size.googleCard}`,
-							`${Size.fluid}`,
+							`${adSizes.outOfPage.toString()}`,
+							`${adSizes.empty.toString()}`,
+							`${adSizes.halfPage.toString()}`,
+							`${adSizes.outstreamMobile.toString()}`,
+							`${adSizes.mpu.toString()}`,
+							`${adSizes.googleCard.toString()}`,
+							`${adSizes.fluid.toString()}`,
 						].join('|')}
 						data-desktop={[
-							`${Size.outOfPage}`,
-							`${Size.empty}`,
-							`${Size.mpu}`,
-							`${Size.googleCard}`,
-							`${Size.video}`,
-							`${Size.outstreamDesktop}`,
-							`${Size.outstreamGoogleDesktop}`,
-							`${Size.fluid}`,
-							`${Size.halfPage}`,
-							`${Size.skyscraper}`,
+							`${adSizes.outOfPage.toString()}`,
+							`${adSizes.empty.toString()}`,
+							`${adSizes.mpu.toString()}`,
+							`${adSizes.googleCard.toString()}`,
+							`${adSizes.video.toString()}`,
+							`${adSizes.outstreamMobile.toString()}`,
```

Comment:
Is this meant to be mobile-sized on desktop?

Reply:
Woopsy, this was a typo. Good spot. Fixing now..